### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Alljobs_Keyboard:Colour contrast ratio is low when we select the steps table.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5914]
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.4.3; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name:colour contrast analiser.
**MAS Reference:**MAS1.4.3
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Navigate to the SQL agent tab->Jobs->Click on any the jobs in the table.
4.Navigate inside the table steps table.
5.Observe the colour contrast ratio of the table when selected.
**Actual:**
colour contrast ratio is less then expected when we select the steps table.
**Expected:**
It should be properly visible when we select the table or de select the table.
**User Impact:**
User with visiual disablities will find it difficult to read the table.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.4.3
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**
[5914 _AccessibilityA11y_AzureDataStudio_AllJobs_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3261322/5914._AccessibilityA11y_AzureDataStudio_AllJobs_Keyboard.pptx)
**Does this issue occur when all extensions are disabled?: Yes**
